server/package.json

```diff
@@ -25,7 +28,11 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
+    "eslint": "^4.10.0",
+    "eslint-config-prettier": "^2.7.0",
```

Comment:
So I think doing the `plugins: ['prettier']` adds the plugin, but do support the rules you gotta mock the extends and rules section like this:

```
{
  "plugins": [
    "prettier"
  ],
  "rules": {
    "prettier/prettier": ["error", {"singleQuote": true}]
  },
  "extends": [
    "react-app",
    "prettier"
  ]
}
```